docs/dev.rst

```diff
@@ -47,38 +71,52 @@ To bring down your platform's containers, simply run::
 
   tutor dev stop
 
-
 Starting the platform back up
 -----------------------------
 
-Once you have used ``launch`` once, you can start the platform in the future with the lighter-weight ``start`` command, which brings up containers but does not perform any initialization tasks::
+Once you have used ``launch`` once, you can start the platform in the future with the lighter-weight ``start -d`` command, which brings up containers *detached* (that is: in the background), but does not perform any initialization tasks::
+
+  tutor dev start -d [--mount=./edx-platform]
+
+If you prefer to run containers *attached* (that is: in the foreground, your current terminal), you can omit the ``-d`` flag::
+
+  tutor dev start [--mount=./edx-platform]
 
-  tutor dev start     # Run platform in the same terminal ("attached")
-  tutor dev start -d  # Or, run platform the in the background ("detached")
+When running containers attached, you can stop the platform with ``Ctrl+c``, or switch to detached mode using ``Ctrl+z``.
+
+Finally, you can always start your platform with ``launch``, even if you have already run it in the past. It will take longer, but it will ensure that your config is applied, your database is provisioned, your plugins are fully initialized, and (if mounted) your local edx-platform is set up. If you include the ``--pullimages`` flag it will also ensure that your container images are up-to-date as well::
```

Comment:
I understand why you removed the `launch is idempotent` paragraph, but I'd like to keep the part where we reassure the users that it's safe to run `launch` multiple times. For instance:

> Running `launch` is always safe, even if it was already run it in the past. It will take longer...